xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1537,7 +1651,7 @@ Otherwise, the token value must be accessed from <xref:System.Text.Json.Utf8Json
         <ReturnType>System.ReadOnlySpan&lt;System.Byte&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the value of the last processed token as a ReadOnlySpan&lt;byte&gt; slice of the input payload. If the JSON is provided within a ReadOnlySequence&lt;byte&gt; and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, the <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
+        <summary>Gets the raw value of the last processed token as a ReadOnlySpan&lt;byte&gt; slice of the input payload. If the JSON is provided within a ReadOnlySequence&lt;byte&gt; and the slice that represents the token value fits in a single segment, then <see cref="P:System.Text.Json.Utf8JsonReader.ValueSpan" /> contains the sliced value since it can be represented as a span. Otherwise, <see cref="P:System.Text.Json.Utf8JsonReader.ValueSequence" /> contains the token value.</summary>
```

Comment:
Here again, everything from "If the JSON" on could be moved to the Remarks section.

Reply:
I could do that, particularly because this is getting lengthy. However, the concern of putting it in remarks is that it won't show up in IntelliSense, and currently this information highlights which property to use in which case.

Is it worth instead adding a "see also" in its place? Or maybe I can condense it a bit and put the details in the remarks.

![image](https://user-images.githubusercontent.com/6527137/64568391-773d7f80-d30f-11e9-89e7-8db5a5221b7e.png)